### DISCUSSION TITLE
What does "Existing Prompt Contents" mean?

### DISCUSSION THREAD

**Discussion by dathide** [d8ahazard/sd_dreambooth_extension#289] [Q&A] [answered]
In the training tab, there are four options for "Existing Prompt Contents". Is this about the Instance Prompt or the Class Prompt? For example, if my Instance Prompt is "photo of zkz dog" and my Class Prompt is "photo of dog", which option should I select for "Existing Prompt Contents"?

Edit: The four options are Description, Instance Token + Description, Class Token + Description, and Instance Token + Class Token + Description

**Comment by d8ahazard** [comment, marked as answer]
This is regarding prompts on existing images. You can caption your instance images, and depending on how you caption them, set this to match, and then have the program automagically swap class/subject in subsequent prompts automatically.

If you're just supplying vanilla instance/class prompts, this can be ignored. It's only relevant if using [filewords] in prompts.

**Reply by dathide** [reply]
Thanks, interesting. Is there a significant benefit to adding captions? If so, is there a guide you recommend for captioning instance images?
